clustered/server/build.gradle

```diff
@@ -22,19 +22,15 @@ sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
 dependencies {
-  compile ("org.terracotta:statistics:$parent.statisticVersion") {
-    exclude group:'org.slf4j', module:'slf4j-api'
-  }
   compile project(':clustered:common'), "org.slf4j:slf4j-api:$parent.slf4jVersion"
-  compile group: 'org.terracotta', name: 'offheap-resource', version: parent.offheapResourceVersion
+  compile("org.terracotta:offheap-resource:$parent.offheapResourceVersion") {
+    transitive = false
+  }
   compile group: 'org.terracotta', name: 'offheap-store', version: parent.offheapVersion
   compile group: 'org.slf4j', name: 'slf4j-api', version: parent.slf4jVersion
   compile("org.terracotta.management:monitoring-service-api:$parent.managementVersion") {
     transitive = false
   }
-  compile ("org.terracotta:statistics:$parent.statisticVersion") {
```

Comment:
removed since statistics is included into management-common and used in management libs